example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -72,34 +72,27 @@
         name: scylla-server
         state: stopped
 
-- name: Restore schema
-  hosts: all[0]
-  become: true
-  gather_facts: false
-  tasks:
-    - name: Remove other seeds
+    - name: Temporarily make the first node a seed
```

Comment:
Why do you need this? 
AFAIU the procedure is supposed to be as follows:

1) Bootstrap the new clusters when all nodes are seeders with `initial_tokens`.
2) Upload the data.
3) Since you upload it to the `data` directory you then restart the service and that's it.

You can change the `seeders` to whatever configuration after that following https://docs.scylladb.com/kb/seed-nodes/#can-i-change-the-seed-node

However you don't have to.
Here you seem to change seeds configuration to have a single (first) node as a seeder.
Since the cluster has already bootstrapped (I didn't realize it before and was referring bootstrapping all the time) you can restart scylla on all nodes at once.

It would also be nice of you to wait for 9042 at the end of the procedure.

BTW, then you restart the service after uploading sstables to the data directory I have a few questions:
1) Will systemd scylla-server service start complete before resharding is over? 
2) With 9042 become available before the resharding is over? I believe it's not. And in that case waiting for 9042 becomes more than "being nice".

Reply:
> Why do you need this?

I experimented and it's true that it is not needed. BUT
It's an optimization, if I don't do that seeds get into some backoff loop and finally boot.

```
May 18 11:22:08 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (215 seconds passed)
May 18 11:22:09 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (216 seconds passed)
May 18 11:22:10 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (217 seconds passed)
May 18 11:22:11 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (218 seconds passed)
May 18 11:22:12 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (219 seconds passed)
May 18 11:22:13 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (220 seconds passed)
May 18 11:22:14 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (221 seconds passed)
May 18 11:22:15 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (222 seconds passed)
May 18 11:22:16 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (223 seconds passed)
May 18 11:22:17 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (224 seconds passed)
May 18 11:22:18 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (225 seconds passed)
May 18 11:22:19 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (226 seconds passed)
May 18 11:22:20 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (227 seconds passed)
May 18 11:22:21 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (228 seconds passed)
May 18 11:22:22 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (229 seconds passed)
May 18 11:22:23 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (230 seconds passed)
May 18 11:22:24 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (231 seconds passed)
May 18 11:22:25 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (232 seconds passed)
May 18 11:22:26 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (233 seconds passed)
May 18 11:22:27 ip-172-31-0-12.eu-central-1.compute.internal scylla[14395]:  [shard 0] gossip - Connect seeds again ... (234 seconds passed
```

If I do that all steps are snappy.

Reply:
Additional twist is that in Scylla Cloud all nodes are seeds at least in my case (6 nodes each is a seed).